.circleci/config.yml

```diff
@@ -65,11 +67,12 @@ commands:
       - run:
           name: Set cargo Environment
           command: |
+            # https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#features
+            export CARGOFLAGS=$(cat cargo-flags)
```

Comment:
I did the same thing with the cargo-flags as the cargo-toolchain, at least until we have an xbuild.